src/alist.c

```diff
@@ -1022,3 +1022,17 @@ void alist_iirf(
     dram_store_u16(hle, (uint16_t*)&ibuf[(index-1)&3], address+10, 1);
 }
 
+/* Perform a clamped gain, then attenuate it back by an amount */
+void alist_overload(struct hle_t* hle, uint16_t dmem, int16_t count, int16_t gain, int16_t attenuation)
```

Comment:
```suggestion
void alist_overload(struct hle_t* hle, uint16_t dmem, int16_t count, int16_t gain, uint16_t attenuation)
```

The ucode uses vmudh (s16 x s16) then vmudm (s16 x u16), so I think that attenuation should be unsigned instead.